frontend/packages/shared/src/enums/TopBarMenu.ts

```diff
@@ -1,11 +1,12 @@
 export enum TopBarMenu {
   About = 'top_menu.about',
+  Access = 'access_control',
```

Comment:
Not sure if this should be there? Access control is in settings modal and not top bar menu?🤔

Reply:
Yes it was there before and not used, so removed it to begin with, but I was not sure so put it back, but if you also mean that it is not suppose to be there I will remove it 😎

Reply:
Yes, this does not belong in this enum 😅 Is it used somewhere?

Reply:
It looks like not, but it is scary to trust Rider 100% 😬 But I guess the tests should capture if it is actually used :p